pkg/node/fetchBalance.go

```diff
@@ -0,0 +1,31 @@
+package node
+
+import (
+	"fmt"
+	"strconv"
+)
+
+type Balance struct {
+	Candidate float64
+	Final     float64
+}
+
+// FetchBalance returns as a float64 the candidate balance and final balance of an address.
+func FetchBalance(client *Client, address string) (*Balance, error) {
+	addressDetails, err := Addresses(client, []string{address})
+	if err != nil {
+		return nil, err
+	}
+
+	candidate, err := strconv.ParseFloat(addressDetails[0].CandidateBalance, 64)
```

Comment:
We should not use floating points for balance & token amounts.
My proposition would be to use this library:
https://pkg.go.dev/github.com/shopspring/decimal

Reply:
I approve that!